usagov/usagov-2021
# Usagov 1878 clear announce

## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-1878

## Description


Announce that selections were cleared when using a screen reader.

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions


Need a screen reader like NVDA active while using the benefits search page.


### Change Requirements



### Validation Steps

- There should be no visual changes to how the search works.
- Screen reader should announce that your selections were cleared when you uncheck the "Select All" button (check once to toggle all on, then again to uncheck all)
- Screen reader should announce that your selections were cleared when you press the "Clear Selectoins" button IF you have one or more categories checked.
- Should also read the Error message if you submit selections with none checked.

## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
